src/ListView/ListView.stories.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { storiesOf, action } from '@kadira/storybook'
+import { withKnobs, boolean } from '@kadira/storybook-addon-knobs'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import MockListManager from './__mocks__/MockListManager'
+import ListView from './ListView'
+import ListViewItem from './ListViewItem'
+import ListViewItemContainer from './ListViewItemContainer'
+import ListViewItemExpansion from './ListViewItemExpansion'
+
+const stories = storiesOf('ListView', module)
+stories.addDecorator(withKnobs)
+stories.addDecorator(
+  defaultTemplate({
+    title: 'List View',
+    documentationLink: 'http://www.patternfly.org/pattern-library/content-views/list-view/'
+  })
+)
+
+stories.addWithInfo(
+  'basic list with expansion',
+  `This is the List View with basic expansion.`,
+  () => <MockListManager />
+)
+
+stories.addWithInfo(
+  'basic list with expansion knobs',
+  `This is the List View with basic expansion.`,
+  () => {
+    const firstLevelExpanded = boolean('First Level Expanded', false)
+    const secondLevelExpanded = boolean('Second Level Expanded', false)
+    return (
+      <ListView>
+        <ListViewItem isActive={firstLevelExpanded}>
+          <ListViewItemContainer
+            isExpansionItem
+            itemClicked={action('first level itemClicked')}
+            isActive={firstLevelExpanded}
+          >
+            First Level
+          </ListViewItemContainer>
+          <ListViewItemExpansion isActive={firstLevelExpanded}>
```

Comment:
IMHO if its a component internal state (e.g isExpanded ) is perfectly fine, it should have a default prop (open or closed) and should be override able via prop and internal state (e.g. when a user expands it).

this will allow a developer to care (or not care) for the initial state himself.

Lastly, I also agree with @matanwerbner and @waldenraines, this component is not easy to use, and since many times a list view is not expandable, maybe we should consider splitting it to two distinct components that can be combined?